### PR TITLE
TeamProwlarr.Prowlarr version 0.1.0.447

### PR DESCRIPTION
**Pull request by alteredpark**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17768)

### CODE REVIEW

File: manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 0.1.0.447
```
Comment (OfficialEsco) [inline manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.installer.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

File: manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 0.1.0.447
+Installers:
+- Architecture: x64
```
Comment (OfficialEsco) [inline manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.installer.yaml:7]:
```suggestion
- Architecture: x86
```

File: manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 0.1.0.447
```
Comment (OfficialEsco) [inline manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

File: manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 0.1.0.447
+PackageLocale: en-US
+Publisher: Team.Prowlar
```
Comment (OfficialEsco) [inline manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.locale.en-US.yaml:7]:
```suggestion
Publisher: Team Prowlar
```

File: manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 0.1.0.447
```
Comment (OfficialEsco) [inline manifests/t/TeamProwlarr/Prowlarr/0.1.0.447/TeamProwlarr.Prowlarr.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Here we go again 🤦‍♂️
image

**Comment by mikeparkie** [comment]
@OfficialEsco yep, understood now, I know that for next time. I'll re-raise if the devs update the packages

File: manifests/t/TeamProwlarr/Prowlarr/0.1.0/TeamProwlarr.Prowlarr.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: TeamProwlarr.Prowlarr
+PackageVersion: 0.1.0
+PackageLocale: en-US
+PackageUrl: https://prowlarr.com
+Publisher: TeamProwlar
```
Comment (OfficialEsco) [inline manifests/t/TeamProwlarr/Prowlarr/0.1.0/TeamProwlarr.Prowlarr.locale.en-US.yaml:6]:
```suggestion
Publisher: Team Prowlar
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
D:

**Review by OfficialEsco — APPROVED** [review]
🎉

**Pull request merged**